packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md

```diff
@@ -17,13 +17,13 @@ This page provides the API rate limits for management activities, which is part
 > * You can expand Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
 >
 
-> We enforce the following **per-minute** limits.
+We enforce the following **per-minute** limits:
 
 | Action and Okta API Endpoint                                                                                                       | Developer (free) | Developer (paid) | One App | Enterprise | Workforce Identity    |
 | ---------------------------------------------------------------------------------------------------------------------------------- | ----------------: | ----------------: | -------: | ----------: | ---------------------: |
 | **Cumulative rate limit**                                                                                                          | **980**          | **5,200**        | **5,200**| **5,200** | **7,000**             |
-| **Create or list applications:**<br>`/api/v1/apps` except `/api/v1/apps/${id}`                                                     | 20               | 100              | 100     | 100        | 100                   |
-| **Get, update, or delete an application by ID:**<br>`/api/v1/apps/${id}` only                                                      | 100              | 600              | 600     | 600        | 500                   |
+| **Create, list, or other application actions:**<br> Calls to the `/api/v1/apps` endpoint and all other endpoints of the form `/api/v1/apps/*` including `/api/v1/apps/${id}/*`                                                     | 20               | 100              | 100     | 100        | 100                   |
+| **Get, update, or delete an application by ID:**<br> Explicit calls to the application instance: `/api/v1/apps/${id}/`                                                    | 100              | 600              | 600     | 600        | 500                   |
```

Comment:
The ending slash was left dangling `/api/v1/apps/${id}`
![image](https://user-images.githubusercontent.com/83234004/199391609-a6866e07-750d-4717-8ca6-c3b0973e0982.png)

Reply:
Thanks, good catch. Will update.